smartseq2_single_sample/tasks/hisat2.wdl

```diff
@@ -5,16 +5,18 @@ task HISAT2PE {
   File fq2
   String ref_name
   String output_name
-  
+  String samplename
```

Comment:
Maybe a space here: `sample_name` for consistency.

Reply:
changed